parser_library/src/workspaces/workspace.cpp

```diff
@@ -664,22 +642,18 @@ void workspace::process_processor_group(
         }
 
         auto lib_local_opts = get_library_local_options(lib, proc_groups, pgm_config);
+        auto rl = transform_to_resource_location(*lib_path, location_);
+        rl.join(""); // Ensure that this is a directory
 
-        if (auto [rl, has_wildcards] = construct_and_analyze_resource_location(*lib_path, location_); !has_wildcards)
-        {
-            rl.join(""); // Ensure that this is a directory
+        if (auto first_wild_card = rl.get_uri().find_first_of("*?"); first_wild_card == std::string::npos)
```

Comment:
wouldn't the characters be url-encoded at this point?

Reply:
Not right now since `?` and `*` characters are ignored during the encoding process. 

Originally, I assumed that since Windows doesn't allow these characters to be present in the filename it is safe to ignore those during encoding but Linux seems to allow it... I'll try to make some test cases for this as well